### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by davidbelius**
The basic example as written produces an error with recently installed versions of python and pytorch (python 3.7.1 and pytorch 1.0.1), because the numpy *_y arrays are created as type int32 which get converted to torch IntTensors, while the cross-entropy loss of pytorch expects LongTensor. This fixes the error by creating the numpy arrays with type int64. See https://pste.eu/p/2N5f.html.

### CODE REVIEW

**Comment by freud14** [comment]
Hi,
Your link doesn't work. Are you it is necessary? Because I just tried to setup a Python 3.7.1 with PyTorch '1.0.1.post2' and it seems to work fine.
Thank you.
Frédérik

**Comment by davidbelius** [comment]
Hi,

Sorry about the link. Attached instead. With the latest Anaconda versions
of Python and pytorch the problem is there. Maybe it's an issue that only
affects pytorch 1.0.1 and was fixed in post2, I don't know!

Regards,
David

Den fre 15 feb. 2019 kl 15:46 skrev Frédérik Paradis <
notifications@github.com>:

**Comment by freud14** [comment]
Hi,

I don't see any attachment. Are you on Windows? It seems that the problem is on Windows and not Linux. Anyway, could you also make a commit for the doc where the same exact code is present?

Thank you.

Fréédrik

**Comment by davidbelius** [comment]
[int_long_issue.zip](https://github.com/GRAAL-Research/pytoune/files/2871157/int_long_issue.zip)

Second try with the attachments. Indeed I am on Windows 10.
